contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -184,6 +184,15 @@ public enum OperationalScheme {
 	@PositiveOrZero
 	public double advanceRequestPlanningHorizon = 0; // beta-feature; planning horizon for advance (prebooked) requests
 
+	@Parameter
+	@Comment(
+			"Time before reaching a planned dropoff from which it is not allowed to insert new detours for new requests. I.e.," +
+					" if set to 180, then a vehicle will not divert to pickup a new passenger once a boarded passenger is only " +
```

Comment:
Suggestion: "divert to pickup a new passenger" -> "divert to pick up or drop off a new passenger"

Reply:
How would you divert for a drop off of a passenger that hasn't boarded yet? Or do you mean the case where both pickup and dropoff of the new passenger are inserted before the next stop?

Reply:
Yes, that's possible, esp. if someone sets this param to larger values.

Reply:
Done.